packages/backend/migrations/20220228162503_create_incoming_payments_table.js

```diff
@@ -11,6 +11,7 @@ exports.up = function (knex) {
     table.string('state').notNullable()
     table.string('externalRef').nullable()
     table.uuid('connectionId').notNullable()
+    table.string('clientId').notNullable()
```

Comment:
What if we store `grantId` on all Open Payments sub-resources (not just outgoing payments) and store the `clientId` on the grant?